core/templates/pages/about-page/about-page.component.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controllers for the about page.
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service.ts';
+import { WindowRef } from
+  'services/contextual/window-ref.service.ts';
+
+require(
+  'components/common-layout-directives/common-elements/' +
+  'background-banner.directive.ts');
+import { TranslateService } from '@ngx-translate/core';
```

Comment:
Please move angular imports at the top, followed by oppia imports, followed by the require statement.

Reply:
Done